api/scripts/run-docker.sh

```diff
@@ -28,6 +28,9 @@ function serve() {
              ${STATSD_HOST:+--statsd-prefix $STATSD_PREFIX} \
              app.wsgi
 }
+function run_task_processor() {
+    python manage.py waitfordb --waitfor 30 --migrations && python manage.py runprocessor --sleepintervalms 500
```

Comment:
Do we need to also check the analytics db is ready here? If analytics are enabled we will need to wait for the analytics db I think?